File: src/main/java/com/haier/uhome/plugins/action/HttpToolAction.java
package com.haier.uhome.plugins.action;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.Project;
import com.intellij.openapi.wm.ToolWindow;
import com.intellij.openapi.wm.ToolWindowManager;

public class HttpToolAction extends AnAction {

    @Override
    public void actionPerformed(AnActionEvent e) {
        Project project = e.getProject();
        if (null != project) {
            ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow("HttpClient");
            if (null != toolWindow) {
                toolWindow.show();
            }
        }
    }
}

File: src/main/kotlin/com/zcy/plugins/action/HttpToolAction.kt
package com.zcy.plugins.action

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.wm.ToolWindowManager
import com.zcy.plugins.utils.Utils

class HttpToolAction : AnAction() {
    override fun actionPerformed(e: AnActionEvent) {
        val project = e.project
        Utils.findGitPath(project)
        project?.let {
            val toolWindow = ToolWindowManager.getInstance(project).getToolWindow("HttpClient")
            toolWindow?.show()
        }
    }
}